src/util/system.cpp

```diff
@@ -1140,6 +1145,38 @@ void runCommand(const std::string& strCommand)
 }
 #endif
 
+#ifdef HAVE_BOOST_PROCESS
+UniValue runCommandParseJSON(const std::string& strCommand, const std::string& strStdIn)
```

Comment:
Will this be used out of RPC? Otherwise could be moved to src/rpc/util.h?

Reply:
Yes, this be used by the wallet (including from the GUI).